docs/Users_Guide/load_data.rst

```diff
@@ -1,333 +1,725 @@
 
-Generate the XML Specification File
+    - Replace *path-to-METdataio* with the path where the METdataio source code is saved
+    - Note the use of the redirection symbol '<' in the command
+
+Create an XML Load Specification Document
+-----------------------------------------
+
+The XML load specification document contains information about the MET data, database connection information, and the location of data to be loaded.
+This is used by the METdbLoad script *met_db_load.py* to load the MET data. The XML load specification is validated against a schema to check
+that the file is valid.  This validation is necessary to prevent extremely large payloads or recursive
+payloads that can compromise the loading of data.
+The elements in the XML load specification file **must** adhere to the **order specified** by the
+XML schema and conform to size and number of element limitations.
+Click to expand the example XML load specifications and the XML validation schema below.
+
+.. dropdown:: An **example XML specification file** that is **valid**
+
+    For defining data organized by dates using the folder_tmpl element:
+
+    .. literalinclude:: ../../METdbLoad/test/Examples/example_load_specification.xml
+
+.. dropdown:: Another **example XML specification file** that is **valid**
+
+    For specifying a list of input data files using the load_files element:
+
+    .. literalinclude:: ../../METdbLoad/test/Examples/example_load_specific_files.xml
+
+
+
+.. dropdown:: The **XML schema** that is used to **validate the XML specification file**
+
+    .. literalinclude:: ../../METdbLoad/ush/load_specification_schema.xsd
+
+Create the XML Specification File
 ===================================
 
-Copy the *data_loading_config.yaml* file to a secure location in your workspace, as this file will contain the username
-and password to the database. **Do not put this file where it can be read by anyone who should not have access to this
-information.**
+The XML load specification file contains the database connection information, data file location, and other instructions for
+loading the data into the database.
+
+
+Create the  XML load specification file by copying the example specification file
+*METdataio/METdbLoad/test/Examples/example_load_specification.xml* file to the workspace location.
+
+**Do not save this XML specification file where it can be read by anyone who should not have access to this information,
+as this file contains the database password.**
 
 .. code-block:: ini
 
-   cp data_loading_config.yaml /path-to-your-dir/
+   cp $METDATAIO_HOME/METdbLoad/test/Examples/example_load_specification.xml  path-to-your-dir/load_specification.xml
 
-Replace the *path-to-your-dir* with the actual path to where this file is to be saved.
 
-Change directory to the location where the *data_loading_config.yaml* file was copied.
+- $METDATAIO is the path to the location of the cloned or forked METdataio source code.
```

Comment:
```suggestion
- $METDATAIO_HOME is the path to the location of the cloned or forked METdataio source code.
```
I believe this needs to be $METDATAIO_HOME and not $METDATAIO as the path references $METDATAIO_HOME.

Reply:
@bikegeek You should be able to click "Commit suggestion" here if you agree.